src/components/Counter.js

```diff
@@ -0,0 +1,12 @@
+import {useState} from 'react';
+
+export default function Example() {
```

Comment:
yout function name is still Example - i guess this is not the name it should have ;)